main.go

```diff
@@ -53,10 +62,40 @@ type influxDBSample struct {
 	Timestamp time.Time
 }
 
+func (c *influxDBCollector) serve() {
+	buf := make([]byte, MAX_UDP_PAYLOAD)
+	for {
+
+		select {
+		default:
+			n, _, err := c.conn.ReadFromUDP(buf)
+			if err != nil {
+				fmt.Printf("Failed to read UDP message: %s", err)
+				continue
+			}
+
+			bufCopy := make([]byte, n)
+			copy(bufCopy, buf[:n])
+
+			precision := "ns"
+			points, err := models.ParsePointsWithPrecision(bufCopy, time.Now().UTC(), precision)
+			if err != nil {
+				fmt.Printf("error parsing udp packet: %s", err)
```

Comment:
We've a logging package you can use for this.

Can you add a metric to count the udp errors?